From 9c5764832bff1608f06ddf2852d3543cf6b56066 Mon Sep 17 00:00:00 2001
From: Soliman162 <ahmedsliman215@gmail.com>
Date: Wed, 25 Oct 2023 22:39:49 +0300
Subject: [PATCH] edit sshd_config

---
 sshd_config | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/sshd_config b/sshd_config
index 19b7c91..8e3d355 100644
--- a/sshd_config
+++ b/sshd_config
@@ -10,7 +10,7 @@
 # possible, but leave them commented.  Uncommented options override the
 # default value.
 
-#Port 22
+Port 22
 #AddressFamily any
 #ListenAddress 0.0.0.0
 #ListenAddress ::
@@ -30,6 +30,7 @@
 
 #LoginGraceTime 2m
 #PermitRootLogin prohibit-password
+PermitRootLogin yes
 #StrictModes yes
 #MaxAuthTries 6
 #MaxSessions 10
@@ -87,7 +88,7 @@ AuthorizedKeysFile	.ssh/authorized_keys
 #X11Forwarding no
 #X11DisplayOffset 10
 #X11UseLocalhost yes
-#PermitTTY yes
+PermitTTY yes
 #PrintMotd yes
 #PrintLastLog yes
 #TCPKeepAlive yes
